<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LIN51010137" failures="2" tests="4" name="Error" time="0.162" errors="0" timestamp="2024-03-20T10:14:18 IST">
  <testcase classname="testngpractice.DependsOnMethodsDemo" name="openApp" time="0.025"/>
  <testcase classname="testngpractice.DependsOnMethodsDemo" name="login" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testngpractice.DependsOnMethodsDemo.login(DependsOnMethodsDemo.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="testngpractice.DependsOnMethodsDemo" name="search" time="0.0">
    <skipped/>
  </testcase> <!-- search -->
  <testcase classname="testngpractice.DependsOnMethodsDemo" name="closeApp" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testngpractice.DependsOnMethodsDemo.closeApp(DependsOnMethodsDemo.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- closeApp -->
</testsuite> <!-- Error -->
